### ISSUE TITLE
Rasterio Dependency version pinned to 1.1.8. 

### ISSUE THREAD

**Issue by CChisuwo** [DHI-GRAS/terracotta#243]
Hello, i am attempting to install terracotta on python version 3.9.5 using pip. This is of course after installing rasterio 1.2.1 from https://www.lfd.uci.edu/~gohlke/pythonlibs/#rasterio. However, the installation seems to query for rasterio 1.1.8. Any advice or workarounds I can use?

**Comment by dionhaefner** [comment]
You can use the current master of Terracotta:

```bash
$ pip install git+https://github.com/DHI-GRAS/terracotta.git
```

**Comment by dionhaefner** [comment]
Fixed in the current release (`0.7.4`).
